### PR TITLE
Change to allow grouping order to be preserved

### PR DESCRIPTION
**Pull request by adkinsrs**
Addresses https://github.com/theislab/diffxpy/issues/205

### CODE REVIEW

**Comment by davidsebfischer** [comment]
Thanks @adkinsrs! Please tag me as reviewer once you are ready to contribute this!

**Comment by adkinsrs** [comment]
@davidsebfischer sadly I do not seem to have permissions to tag you as a reviewer.

File: diffxpy/testing/utils.py [hunk 1]
```diff
@@ -111,7 +111,8 @@ def parse_grouping(data, sample_description, grouping):
 
 def split_x(data, grouping):
     grouping = np.asarray(grouping)
-    groups = np.unique(grouping)
+    #groups = np.unique(grouping)
```
Comment (davidsebfischer) [inline diffxpy/testing/utils.py:114]: Feel free to take this comment out, will merge after!

**Comment by davidsebfischer** [comment]
> @davidsebfischer sadly I do not seem to have permissions to tag you as a reviewer.

Sorry was not aware of that, I reviewed it now!

**Comment by davidsebfischer** [comment]
Thank you, @adkinsrs!

**Pull request merged**